src/uu/stty/src/stty.rs

```diff
@@ -174,8 +175,9 @@ fn stty(opts: &Options) -> UResult<()> {
         ));
     }
 
+    let borrowed_fd = unsafe { BorrowedFd::borrow_raw(opts.file) };
```

Comment:
I think `opts.file` should be a `File` instead, which `tcgetattr` and `tcsetattr` should accept now because it implements `AsFd`. If that's not possible it could be an `OwnedFd`.

Reply:
Yep, it's possible, though I had to use an enum because we use both `File` and `Stdout`.

Reply:
Sounds great!